docs/configuration/data-validation.md

```diff
@@ -94,17 +94,17 @@ The following colors are supported:
 
 !!! tip "Dynamic Configuration Parameter"
 
-This is a mapping of models to [custom validators](../customization/custom-validation.md) against which an object is evaluated immediately prior to its deletion. If validation fails, the object is not deleted. An example is provided below:
+This is a mapping of models to [custom validators](../customization/custom-validation.md) against which an object is evaluated immediately prior to its deletion. If validation fails, the object is not deleted. The configuration can be carried out in the menu under “System/Configuration History/Custom validators”. 
```

Comment:
> The configuration can be carried out in the menu under “System/Configuration History/Custom validators”.

This is implied by the "dynamic configuration parameter" note above, which appears on many similar parameters.

Reply:
OK I have the connection between the:
!!! tip "Dynamic Configuration Parameter"

and not understood, as there is no link referencing it.
https://demo.netbox.dev/static/docs/configuration/#dynamic-configuration-parameters

It was only through your comment and Google that I found the connection again.

If there was a link to dynamic-configuration-parameters, you could probably understand it better.

I would like to adapt the tip like this:
!!! tip [`Dynamic Configuration Parameter`](./index.md#dynamic-configuration-parameters)

Reply:
Is it ok if I adjust all these tips?

Before: `!!! tip ‘Dynamic Configuration Parameter’`
After: `!!! tip [`Dynamic Configuration Parameter`](./index.md#dynamic-configuration-parameters)`